assistant-app.js

```diff
@@ -2472,7 +2472,7 @@ class AssistantApp {
     }
     const { userStorage } = user;
     if (!userStorage) {
-      error('No userStorage found');
+      debug('No userStorage found');
```

Comment:
Why change this and not line 2470?

Reply:
was originally have line 2470 in FR, but accordingly to @Canain , user object should always exist?

Reply:
I think that this is a critical bug. Basically, I think that the `user` object is passed always. However, currently, actions on google sometimes sends a request without the `userStorage` object. And, when the request doesn't have the userStorage object, this actions-on-google-nodejs sdk is crashed in the `DialogflowApp` constructor by the `error()` call. I guess that many developers have the same problem and cannot use the latest version. I expect that this pull request is merged quickly and a fixed version will be released soon.